packages/studio-plugin/src/ParsingOrchestrator.ts

```diff
@@ -196,6 +204,17 @@ export default class ParsingOrchestrator {
     return this.filepathToFileMetadata;
   }
 
+  private initLayoutNames(): string[] {
```

Comment:
Is there a separate item for parsing layouts?

Reply:
I believe Tom is making one!